Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN yarn release-docker
 
 FROM golang:1.16.5-alpine3.13 as backend
 
-ARG VERSION="development"
```

Comment:
This is intentional, our build_docker_bb would overwrite it.

By doing this we know, if the VERSION is still development, then it's not built via build_docker_bb